### PR TITLE
add new JECs and update code

### PR DESCRIPTION
**Pull request by cheidegg**
from commit dad137e3a148cdf2f3d6d690aeca8293fd2bd8a4
new JEC files AND some code updates
to be merged ON TOP of leptonic EWK SUSY PR #142

### CODE REVIEW

File: TTHAnalysis/cfg/run_susyMultilepton_cfg.py [hunk 1]
```diff
@@ -44,8 +44,14 @@
 # Lepton Skimming
```
Comment (peruzzim) [inline TTHAnalysis/cfg/run_susyMultilepton_cfg.py:44]: This and other modifications to configuration files are also in #142. It should not be the case... if left like this, one will need to be careful to merge correctly.

File: TTHAnalysis/data/json/Cert_271036-283685_13TeV_PromptReco_Collisions16_JSON_NoL1T.txt [hunk 2]
```diff
@@ -0,0 +1,365 @@
+{"273158": [[1, 1279]],
```
Comment (peruzzim) [inline TTHAnalysis/data/json/Cert_271036-283685_13TeV_PromptReco_Collisions16_JSON_NoL1T.txt]: If we really need to add a JSON, we should have only the latest version.

**Review by peruzzim — CHANGES_REQUESTED** [review]
tar.gz files are not needed and should not be added to the repository.
Also, please remove the hidden files in the JEC directory (._Spring16_25nsV8BCD_DATA_L2L3Residual_AK4PFchs.txt etc.).
Some other modifications/additions are duplicate with #142.

**Comment by mariadalfonso** [comment]
@cheidegg can you add the .db file of the Spring16_25nsV8 so that we have a complete set ?

**Comment by mariadalfonso** [comment]
@cheidegg 
the JSON file can be remove, and maybe point to the original in your cfg  i.e. ?
/afs/cern.ch/cms/CAF/CMSCOMM/COMM_DQM/certification/Collisions16/13TeV/Cert_271036-276811_13TeV_PromptReco_Collisions16_JSON_NoL1T.txt

**Review by peruzzim — APPROVED** [review]

**Comment by mariadalfonso** [comment]
@peruzzim @gpetruc @cheidegg
I would suggest to wait to merge this PR, as the V8 JEC are going to be depreccted  (as buggy)
V10 are going to be releases very very soon 
https://github.com/cms-jet/JECDatabase/tree/master/textFiles

**Comment by peruzzim** [comment]
@mariadalfonso as the analyzers have already shared this branch and used it (even if JECs are buggy) for some internal results, I would propose to merge it and ensure that V8 is not used by default

**Comment by gpetruc** [comment]
Hi Constantin, can you quickly fix the conflict?

**Comment by gpetruc** [comment]
Uhm, after some look, I think this PR is out of sync with some fixes, e.g. the first modification of https://github.com/CERN-PH-CMG/cmgtools-lite/pull/142/commits/e4ba1a9a5f9853e5f844fe239d2154d0ae942868 or the changes and the change of way the run-dependent JECs are configured.

**Comment by gpetruc** [comment]
I will merge this afternoon manually but leave the default JECs to V6 for susyCore.
I leave to you to fix in a future local PR the running with analysis == susy.

**Pull request merged**